### DISCUSSION TITLE
Sort by Date in Obok dialog

### DISCUSSION THREAD

**Discussion by brendontowle** [noDRM/DeDRM_tools#257] [Ideas]
I read a lot, and so I'm adding 1-2 books/week through the Obok plugin. It would really make my life a lot easier if I could sort the books in the dialog by added date, and just pick the top book(s) off of that sort.

Any possibility of this?

**Comment by ElleKayEm** [comment]
I agree this would be nice.  In the meantime, the current release doesn't add duplicates.  Of course, if you've edited a title at all (i.e. removed a subtitle like "A Novel"), it won't realize it's a duplicate.

**Reply by brendontowle** [reply]
And, lucky me, I edit the titles of almost everything I read. :-(
